roles/collector_pd/tasks/main.yml

```diff
@@ -13,6 +13,12 @@
 - include_tasks: collect_log.yml
 - include_tasks: collect_config.yml
 
+- name: collect PD information
```

Comment:
inventory_hostname is not ip addr if we set a alias for pd instance in `inventory.ini`, like `pd1 ansible_host=172.16.20.72`. 
refer: https://github.com/pingcap/tidb-ansible/blob/master/roles/pd/tasks/check_certs.yml#L21